### ISSUE TITLE
Cannot load anything from remote storage / "Validation failed"

### ISSUE THREAD

**Issue by stueja** [Laverna/laverna#515]
I am using the "self-hosted" webapp on nginx together with [php-remote-storage](https://github.com/fkooman/php-remote-storage).
I can successfully connect to the remote storage, and notes, notebooks, tags and profiles are being written to the storage (as visible in the file system).

However, if I close the browser and open it again of if I use another browser or another browser instance and return to Laverna, it will not show anything. Just blank.

Errors in the console (here: Google Chrome) are like
```
Validation failed:notes ["title"]   main.js:157
```
with details as in this screenshot:
![2016-07-22-124418-1](https://cloud.githubusercontent.com/assets/9169222/17055218/9f9fdcbe-500b-11e6-8869-3b91425def6f.jpg)

I am not even sure if those both topics (Validation failed and can't load anything) are related.
Main question: how can I make Laverna show my stored notes?0

**Comment by joel-porquet** [comment]
I have the same issue.

php-remote-storage is installed on my personal server and I use the laverna app from the official website. Authentication works but notes are not displayed in the web app. The console shows:

```
Validation failed:notes Array [ "title" ]main.js:157:2004
r<.saveRaw/<()main.js:157
o()q.js:28
h.prototype.then/</<()q.js:28
h/r.promiseDispatch()q.js:28
h.prototype.then/<()q.js:28
n()q.js:28
t()
```

It seems like php-remote-storage returns the notes to the web app, according to the log of my server:

```
"GET /joel/laverna/ HTTP/1.1" 304 0 "https://laverna.cc/app/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71 Safari/537.36"
"GET /joel/laverna/notes-db/notes/ HTTP/1.1" 304 0 "https://laverna.cc/app/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71 Safari/537.36"
"OPTIONS /joel/laverna/notes-db/notes/753fa5ac-d233-17fa-a330-1e680d76af65 HTTP/1.1" 200 0 "https://laverna.cc/app/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71 Safari/537.36"
"GET /joel/laverna/notes-db/notes/753fa5ac-d233-17fa-a330-1e680d76af65 HTTP/1.1" 304 0 "https://laverna.cc/app/" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/54.0.2840.71 Safari/537.36"
```

Any idea why it doesn't work?

**Comment by picarsite** [comment]
I have the same problem. Any solution?

**Comment by joel-porquet** [comment]
https://github.com/fkooman/php-remote-storage/issues/50

**Comment by anmol26s** [comment]
Hi @stueja 
Have you able to fix this? I am having the same issue.

**Comment by stueja** [comment]
@anmol26s No, sorry, weren't able and haven't cared for this issue since. Use another product to store notes.

**Comment by blubbsy** [comment]
Hi,

i'm facing exactly the same problem and couldnt find any solution to fix it. everything appears to work correctly, but i cant load my notes from another instance. this happens also in other web apps (e.g. litewriter).
